### PR TITLE
Add route that takes body parameters

### PR DESCRIPTION
**Pull request by valentino-amadeus**
#### What is the purpose of this pull request?
Add initializing route that takes the initialization parameters (`InitializingWorkspace`, `Hours` (of initial stage) and (initial) `Proportion` (of traffic)) from the request's body. 

#### What problem is this solving?
The parameters are currently passed through the request's url. With this new feature, we can create further features with improved ease - without having to create new routes at every option we add to the initialization process. 

#### How should this be manually tested?
By calling the new route and checking whether the test is initialized successfully.

#### Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: node/utils/BodyParser/getRequestParams.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+}
+
+const checkForExpectedFields = (object: object) => {
+    for (let idx = 0; idx < expectedFields.length; idx++) {
+        const field = expectedFields[idx]
+        if (!(field in object)) {
+            throw new Error(`Error getting request's parameters: make sure to set the ${field} field`)
+        }
+    }
+}
```
Comment (tergol1) [inline node/utils/BodyParser/getRequestParams.ts:19]: Don't we have a library for this?

**Review by tergol1 — APPROVED** [review]
LGTM!
Should we remove de `.DS_Store` fike?

**Comment by valentino-amadeus** [comment]
> LGTM!
> Should we remove de `.DS_Store` fike?

Yes, we should hehe. Is is removed here: https://github.com/vtex/ab-tester/pull/67

File: node/utils/BodyParser/getRequestParams.ts (hunk 1 above)
Comment (valentino-amadeus) [reply node/utils/BodyParser/getRequestParams.ts:19]: I confess I didn't look for a library. You're talking about a library of `node-vtex-api`?

**Pull request merged**